region/brinstar/red.json

```diff
@@ -1176,7 +1205,14 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": ["h_canPassBombPassages"]
+                  "requires": [
+                    "h_canPassBombPassages",
+                      {"or": [
+                        "HiJump",
+                        "h_canFly",
+                        "canWalljump"
```

Comment:
SpringBall Jumps work here, as well as a 33-frame shinespark coming in from node 1 (needs about 60 remaining frames).